.github/workflows/recurrent-checks.yml

```diff
@@ -2,7 +2,7 @@ name: CI check
 
 on:
   schedule:
-    - cron: "1 01,13 * * *" #lets run this job every 12 hours.
+    - cron: "1 01 * * *" #lets run this job every 12 hours.
```

Comment:
This is every day at 01:01 (I think!). Can we update either the cron to be every 12 hours or the comment to say it runs nightly?

Reply:
true